themes/ddbasic/sass/components/panel/class-panel.scss

```diff
@@ -778,6 +778,26 @@
       }
     }
   }
+
+  .views-widget-filter-term_node_tid_depth {
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-909

Reply:
Legit (except for the naming) - see comment in `modules/ding_event/ding_event.module`.

Reply:
To match the changes in the key - https://github.com/easySuite/ding2/pull/664/files#r517941753

Reply:
I don't see a reason to change this key.

Reply:
How come? It was changed to match the styles of the filter whose key was changed.

Reply:
I'm talking about the key you are linking to. I don't see a reason for changing that key name.

Reply:
I don't think I understand. The key was changed here - https://github.com/easySuite/ding2/commit/99827deab74dcd9fdc00ff470e734c52276a0473#diff-910ba0361c20fb0a154046bcf13046d70e01eae0bd1ff940848ec844352625bbR328
So did the css class to match the changes.

Reply:
Okay, gotcha; it's changes in main distro. We should look into why these changes are here.
When I look at diff from ding2: https://github.com/ding2/ding2/compare/7.x-6.1.0...easySuite:7.x-6.1.0+easy I see changes in `modules/ding_event/ding_event.module` that origins from https://github.com/easySuite/ding2/pull/498 and  https://github.com/easySuite/ding2/pull/502, however I do not understand what these changes do. I see no changes at all to the Event overview (/arrangementer) page when comparing ding2 and EO.

Reply:
I'mg gonna remove this, since this already exists - https://github.com/easySuite/ding2/blob/development/themes/ddbasic/sass/components/panel/class-panel.scss#L684